### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by krish9d** [silviaruiz44/HRAnalytics#7]
This looks to be a very interesting problem, and probably has real world applications. You seem to have already identified a good number of 'predictors' which make sense for your project such as (age, department etc).

You seem to have done an excellent job with initial data exploration, transformation and analysis. Your project objective and the report are also very clear. You also seem to have a strong understanding of what the next steps in your projects are, which is a good sign.

I would be concerned about the amount of data you have. Training data of 3200, examples is not very much and it is very easy to overfit. Also you say that you would like to use simpler models, so that you do not overfit. One concern would be the simpler model, you plan to use, might miss out on many subtle trends. Another concern would be your error reporting. You could, just by predicting that no-one leaves the company attain a high accuracy but what matters is your false positive and false negative rates.
